pytest_xvfb.py

```diff
@@ -130,8 +131,12 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    if config.getoption('--no-xvfb') or not xvfb_available():
+    if config.getoption('--no-xvfb'):
         config.xvfb = None
+    elif not xvfb_available():
+        config.xvfb = None
+        warnings.warn(
+            'Pytest-Xvfb could not find Xvfb installation. Please see that it is installed.')
```

Comment:
Good points. I agree with the warning statement change, and I completely forgot about other operating systems so I'll add a condition with sys.platform. Should I also remove the warning for unset displays since the tests will fail anyway? Regarding --no-xvfb, the warning does not print if --no-xvfb is given as a flag as it's captured by the first if statement.